### ISSUE TITLE
Top menu doesn't work without JS

### ISSUE THREAD

**Issue by Y-Less** [mdn/yari#2149]
I say "doesn't work without JS", however, I've not actually confirmed that it does work with JS, so maybe this is just "Top menu doesn't work".  Currently, clicking on "Technologies", "References & Guides", and "Feedback" does absolutely nothing.

As maybe the most important web development resource, progressive enhancement and compatibility should be a number one priority.  I know many devs forget (or more likely just don't care) that not everyone has the cutting-edge machines and software that they do when they make websites; please don't be like them - lead by example (though I'm pleased by the existing percentage that doesn't require JS for nothing special, much higher than many plain text site's working percentage of 0).

**Comment by Y-Less** [comment]
Yes.  I can see those three headers, but clicking them does nothing at all.

**Comment by mattiapontonio** [comment]
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details

And details in the example works?

**Comment by schalkneethling** [comment]
> Yes. I can see those three headers, but clicking them does nothing at all.

It is definitely sad that the menu does not work without JavaScript. There are actually a couple of things that currently does not work without JavaScript enabled. @Y-Less rest assured that we are having an internal discussion around this entire topic and we appreciate your feedback 🙏

**Comment by peterbe** [comment]
Do you need the menu to navigate? I appreciate that the links aren't that important but it would be nice to be able to search. 
For example, amazon.com's awesome search widget and other things don't work if you disable JavaScript. You can at least submit your search and you can click to sign in and click to view your cart etc. 
<img width="1234" alt="Screen Shot 2020-12-18 at 4 49 30 PM" src="https://user-images.githubusercontent.com/26739/102664633-34449b80-4151-11eb-90da-5e52b4460fa6.png">

Mind you, they probably have as many people working on the nav, as we have in the entire team :)

**Comment by Y-Less** [comment]
I didn't even know Amazon had anything other than a basic text input for their search bar, but it has always worked fine for me.

**Comment by schalkneethling** [comment]
Duplicate of https://github.com/mdn/yari/issues/2439

**Comment by Y-Less** [comment]
> 
> 
> Duplicate of #2439

This was first by almost a month; surely that makes #2439 the duplicate?
